texts/depositing/overview.yml

```diff
@@ -1,3 +1,14 @@
 title: Depositing performance
-description: You have about {{amount}} non-compliant works.
+description: >
+  {{#isLoading}}
+     We are collecting data. It should appear anytime soon.
+  {{/isLoading}}
+  {{^isLoading}}
+    {{#count}}
+       You have about {{amount}} non-compliant works.
+    {{/count}}
+    {{^count}}
+       We couldn't find any data about depositing performance.
+   {{/count}}
+  {{/isLoading}}
```

Comment:
`@oacore/texts` was designed for a content manager to be able edit texts with minimal involvement of developer. I do not think that somebody not-technical would learn another technology just because of this. Such text is not clear enough to easily edit by other people.

To be fair, I understand it only because I know the task. I am not familiar with Mustache enough to know that it's conditional rendering.